aws-android-sdk-s3/build.gradle

```diff
@@ -7,10 +7,22 @@ android {
     defaultConfig {
         minSdkVersion 9
         targetSdkVersion 29
+        // Required when setting minSdkVersion to 20 or lower
+        multiDexEnabled true
+    }
+
+    compileOptions {
+        // Flag to enable support for the new language APIs
+        coreLibraryDesugaringEnabled true
+        // Sets Java compatibility to Java 8
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
     }
 }
 
 dependencies {
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.9'
```

Comment:
`desugar` is set to make try-with-resources statements compatible with Java 8 without setting minSDK.